packages/beacon-node/test/sim/mergemock.test.ts

```diff
@@ -221,6 +255,14 @@ describe("executionEngine / ExecutionEngineHttp", function () {
       );
     }
 
+    // 2. builder and engine blocks are as expected
+    if (builderBlocks < expectedBuilderBlocks || engineBlocks < expectedEngineBlocks) {
+      throw Error(
+        `Incorrect builderBlocks=${builderBlocks} (expected=${expectedBuilderBlocks}) or engineBlocks=${engineBlocks} (expected=${expectedEngineBlocks})`
+      );
+    }
```

Comment:
Split this into two separate ifs and throws

Reply:
done :+1: